pkg/controllers/status/cluster_status_controller.go

```diff
@@ -309,24 +300,42 @@ func (c *ClusterStatusController) initLeaseController(ctx context.Context, clust
 	}()
 }
 
-func getClusterHealthStatus(clusterClient *util.ClusterClient) (online, healthy bool) {
```

Comment:
I don't think we need to put more logic into this function. 
This function just tells the result of this time, and don't have to know how we use this result.